components/AuthorizationForm.js

```diff
@@ -252,14 +252,18 @@ const AuthorizationForm = () => {
   }
 
   return (
-    <div className='wrapper'>
+    <div className='wrapper' onClick={() => {router.back()}}>
```

Comment:
This will look better as a handleClose method - we will get rid of the arrow functions and code duplication

Reply:
stopPropagination could be moved to handleLockPropagination as well :)